doc/admin-guide/plugins/rate_limit.en.rst

```diff
@@ -82,27 +82,27 @@ Examples
 
 This example shows a simple rate limiting of `128` concurrently active client
 transactions, with a maximum queue size of `256`. The default of HTTP status
-code `429` is used when queue is full.
+code `429` is used when queue is full. ::
```

Comment:
Good improvement. 

As a small tweak, can we put the `::` on a line by itself? That seems more common:

- https://raw.githubusercontent.com/apache/trafficserver/master/doc/admin-guide/configuration/transparent-proxy/router-inline.en.rst
- https://raw.githubusercontent.com/apache/trafficserver/master/doc/developer-guide/debugging/debug-builds.en.rst

To name a couple examples. Let me know if you had a reason to put the `::` at the end of the previous line that I'm not aware of.

Reply:
Done. Thanks!